### PR TITLE
Tweaks and clarification for ROVIO config exporter

### LINKED ISSUE #161: rovio config exporter extrinics
There might be a bug in the rovio config exporter.

So here we read T_cam_imu (trafo from imu to camera frame) from the Kalibr config file and invert, giving us T_SC and q_SC (trafo from camera to imu frame, i.e. camera position and orientation in imu frame).

https://github.com/ethz-asl/kalibr/blob/b95f5b83583e0a0cd1b06a4e50b9205a464170ba/aslam_offline_calibration/kalibr/python/exporters/kalibr_rovio_config#L112-L114

Later we assign this to variables qCM and MrMC from the rovio config file.

https://github.com/ethz-asl/kalibr/blob/b95f5b83583e0a0cd1b06a4e50b9205a464170ba/aslam_offline_calibration/kalibr/python/exporters/kalibr_rovio_config#L33-L36

Judging from the notation and also the rovio IJRR paper, it seems that while MrMC matches the input (camera position in imu frame), qCM does not match, as it represents the imu orientation in camera frame (i.e. q_SC inverted).

Is this a bug or did I miss something?

### PR DESCRIPTION
**Pull request by NikolausDemmel**
- clarify conversion of extrinsics (it wasn't wrong, but confusing, see
   #161)
 - fix mapping of distorion model names and parameters (ROVIO expects
  'plumb_bob' instead of 'radtan')
 - add some command line output about which files were written
 - remove spurios newlines in generated files

Fixes #161

### CODE REVIEW

File: aslam_offline_calibration/kalibr/python/exporters/kalibr_rovio_config [hunk 1]
```diff
@@ -11,7 +11,29 @@ import rospkg
...
+        return "plumb_bob", list_fill_with(params_kalibr, 5, 0.0)
+    elif model_kalibr == "equidistant":
+        return "equidistant", list_fill_with(params_kalibr, 4, 0.0)
+    elif model_kalibr == "fov":
+        return "fov", list_fill_with(params_kalibr, 1, 0.0)
+    else:
+        return model_kalibr, params_kalibr
+
+
+def printCameraBlock(camConfig, T_SC, q_CS):
```
Comment (HannesSommer) [inline aslam_offline_calibration/kalibr/python/exporters/kalibr_rovio_config]: Shouldn't this be `t_SC` as well?

**Comment by HannesSommer** [comment]
ok to test

File: aslam_offline_calibration/kalibr/python/exporters/kalibr_rovio_config (hunk 1 above)
Comment (NikolausDemmel) [reply aslam_offline_calibration/kalibr/python/exporters/kalibr_rovio_config]: Shame on me for last minute renaming without testing. The updated version runs for my example. I can squash before merge if needed.

**Comment by HannesSommer** [comment]
Anybody else willing to have a closer look? @ZacharyTaylor ? @schneith ?

**Comment by ZacharyTaylor** [comment]
Had a quick look and lgtm, thanks for this

**Comment by HannesSommer** [comment]
Thanks @ZacharyTaylor ! I'll squash merge it.

**Comment by NikolausDemmel** [comment]
Thanks for quick review and merge!

**Pull request merged**